docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Mint NFTs on Avalanche 🚧
+description: Learn how to start minting NFTs on Avalanche.
+issueUrl: https://github.com/protocol/nft-website/issues/224
+---
+
+# Mint NFTs on Avalanche
+
+This short tutorial will quickly guide you through getting started with an EVM-compatible NFT minting work flow on [Avalanche](https://www.avax.network/)using Node.js REPL. This is quite useful if you are migrating from Ethereum or other EVM-compatible blockchain and wish to reuse your code.
+
+## Avalanche vs Ethereum
+
+Unlike Ethereum, Avalanche is a network of multi-blockchains, one of which runs a fork of Ethereum Virtual Machine (EVM) and compatible with Ethereum.
+
+Avalanche consists of 3 subnets, X-chain, P-chain, and C-chain.
```

Comment:
3 subnets: X...

Reply:
Should it be 4 network, one primary network and 3 subnets ?

Reply:
Is it wierd to write: "... subnets: The X-, P-, and C-chains." @DeveloperAlly @mishmosh

Reply:
> Should it be 4 networks, one primary network, and 3 subnets?
I guess you're right. But here we are mentioning the subnets. The primary network is sort of the bigger one.